proposals/0002-root-signature-in-clang.md

```diff
@@ -554,9 +554,7 @@ The additional semantic rules not already covered by the grammar are listed here
 
 | Property       | Type              | Type detail                                         |
 | -------------- | ----------------- | --------------------------------------------------- |
-| RootFlag       | i32               | D3D12_ROOT_SIGNATURE_FLAG                           |
-| Parameters     | reference to list | list of RootDescriptor/RootConstant/DescriptorTable |
-| StaticSamplers | reference to list | list of StaticSamplers                              |
+| Parameters     | reference to list | list of  D3D12_ROOT_SIGNATURE_FLAG/RootDescriptor/RootConstant/DescriptorTable/StaticSamplers |
```

Comment:
The grammar refers to these collectively as "Root elements".  We may as well just use the same terminology in the metadata format.

Reply:
Updated.